defaults/docker_images/gemini/values_gemini.yaml

```diff
@@ -1,2 +1,2 @@
 gemini:
-  image: 'scylladb/hydra-loaders:gemini-v1.8.6'
+  image: 'scylladb/gemini:dev'
```

Comment:
please use specific image names, it would be hard to track exactly what image was used in runs

Reply:
please update this to official release